integration-tests/src/test/java/com/datastax/oss/driver/mapper/DeleteIT.java

```diff
@@ -56,8 +56,8 @@
 // Do not run LWT tests in parallel because they may interfere. Tests operate on the same row.
 @BackendRequirement(
     type = BackendType.CASSANDRA,
-    minInclusive = "3.0",
-    description = ">= in WHERE clause not supported in legacy versions")
+    minInclusive = "3.4",
+    description = ">= in WHERE clause, and SASI indexes are not supported in legacy versions")
```

Comment:
I'm kinda confused by this.  The old test could test >= without SASI indexes from C* 3.0 on.  We enabled SASI indexes in the C* config in order to support testing against C* 5.0... and as a result we have to bump the minimum version of this test case up to C* 3.4.  That seems weird to me... seems like maybe we should bundle any SASI-related tests elsewhere to avoid changing the max version here.

Reply:
In related news... do we actually need the SASI config changes here?  I couldn't find any other changes in this test case related to the work around C* 5.0 testing support.

Reply:
During 5.0 change, we disabled parallel execution. Tests executed in parallel manner have `DefaultCcmBridgeBuilderCustomizer`, which configures SASI index when needed. That is why test "worked" before on C* versions 4+.

If SASI indexes are disabled, test setup fails:
```
    if (supportsSASI(ccmRule) && !isSasiBroken(ccmRule)) {
      builder.add(
          "CREATE CUSTOM INDEX product_description ON product(description) "
              + "USING 'org.apache.cassandra.index.sasi.SASIIndex' "
              + "WITH OPTIONS = {"
              + "'mode': 'CONTAINS',"
              + "'analyzer_class': 'org.apache.cassandra.index.sasi.analyzer.StandardAnalyzer',"
              + "'tokenization_enable_stemming': 'true',"
              + "'tokenization_locale': 'en',"
              + "'tokenization_skip_stop_words': 'true',"
              + "'analyzed': 'true',"
              + "'tokenization_normalize_lowercase': 'true'"
              + "}");
    }
```
This setup method is declared in the superclass, reused by 16 other tests. Actually, only `SelectCustomWhereClauseIT` needs SASI index, so indeed maybe it is better to add a flag indicating whether SASI index is required by the test.

Reply:
I'd say make it even easier: just overload createStatements() with requiresSasiIndex as false in the default case:

```java
  protected static List<String> createStatements(BaseCcmRule ccmRule) {
    return createStatements(ccmRule, false);
  }
```

Unless I'm missing something that means you only need to modify SelectCustomWhereClauseIT since it's the only one that actually passes in "true" for the SASI create argument.

Reply:
Good for second round of review.